scripts/populate_sample_contributor_data.py

```diff
@@ -253,6 +255,19 @@ def _add_topics_to_classroom(
             'csrf_token': self.csrf_token
         }
```

Comment:
Do we need to drop this (i.e. editing the config property)?

Also, /cc @lkbhitesh07 -- just to check, has your migration removed other cases in the code where we set config properties?

Reply:
Yes, My migration removes the cases where we set the config property and instead, now we update the platform parameters. Currently, my PRs cover the migration for all of the config properties and are in the testing stage except the `CLASSROOM_PAGES_DATA` and `RECORD_PLAYTHROUGH_PROBABILITY`. The `CLASSROOM_PAGES_DATA` will be taken care by @Nik-09 as discussed previously and I will be migrating the `RECORD_PLAYTHROUGH_PROBABILITY` when the classroom migration will be completed, currently, we have a bunch of general tests related to config properties like check-if-the-config-property-works-as-expected and more which requires at least one working config property and these tests helps in code coverage as well. So, after the classroom migration, I will be removing all the dependencies of the config properties from the codebase and will be migrating `RECORD_PLAYTHROUGH_PROBABILITY`.
Thanks

Reply:
Sounds good. Thanks for the detailed answer, @lkbhitesh07!

Reply:
+1 @lkbhitesh07